megamek/src/megamek/client/ui/swing/DeploymentDisplay.java

```diff
@@ -530,10 +530,12 @@ public void hexMoused(BoardViewEvent b) {
             } else if (!isAero && !isWiGE) {
                 // hovers and naval units go on the surface
                 if ((ce().getMovementMode() == EntityMovementMode.NAVAL)
-                        || (ce().getMovementMode() == EntityMovementMode.SUBMARINE)
                         || (ce().getMovementMode() == EntityMovementMode.HYDROFOIL)
                         || (ce().getMovementMode() == EntityMovementMode.HOVER)) {
                     ce().setElevation(0);
+                } else if (ce().getMovementMode().isSubmarine()) {
+                    // submarines have one level above the surface
```

Comment:
below?

Reply:
No, above. Those that are multiple levels in height have one level above the surface (when deployed on top of the water) and the rest below.